fpga/src/main/scala/arty100t/HarnessBinders.scala

```diff
@@ -41,3 +43,43 @@ class WithArty100TDDRTL extends HarnessBinder({
     ddrClientBundle <> port.io
   }
 })
+
+// Uses PMOD JA/JB
+class WithArty100TSerialTLToGPIO extends HarnessBinder({
+  case (th: HasHarnessInstantiators, port: SerialTLPort) => {
+    val artyTh = th.asInstanceOf[LazyRawModuleImp].wrapper.asInstanceOf[Arty100THarness]
+    val harnessIO = IO(port.io.cloneType).suggestName("serial_tl")
```

Comment:
Should be `chiselTypeOf(port.io)` instead of `port.io.cloneType` (otherwise throws an error when the SerialTL clock is an input).